README.md

```diff
@@ -1,59 +1,62 @@
-#  <img src="docs/doctorkafka_logo.svg" alt="DoctorKafka logo" width="48"> &nbsp;&nbsp; Pinterest DoctorKafka
+#  <img src="docs/doctorkafka_logo.svg" alt="DoctorKafka logo" width="48px"> &nbsp;&nbsp; Pinterest DoctorKafka
 
-[![Build Status](https://travis-ci.org/pinterest/doctorkafka.svg)](https://travis-ci.org/pinterest/doctorkafka)
+***ATTENTION: `master` branch now points to DoctorKafka v3 (0.3.x). The legacy version (0.2.x) can be found under the `0.2.x` branch. 0.3.x and 0.2.x versions are INCOMPATIBLE.***
```

Comment:
explain more on what is incompatbile?  

    0.3.x uses yaml based configuration file that is not compatible with 0.2.x. 
    You need to update DoctorKafka configuration to use 0.3.x.